test/bot.test.ts

```diff
@@ -215,6 +218,65 @@ describe('gitpoap-bot', () => {
       ]);
     });
 
+    it('should not create claims for invalid users', async () => {
+      const githubAPIMock = nock('https://api.github.com')
+        // Test that we correctly return a test token
+        .post('/app/installations/29153052/access_tokens')
+        .reply(200, {
+          token: 'test',
+          permissions: {
+            issues: 'write',
+          },
+        })
+
+        // get github login ids
+        .get('/users/test-test-')
+        .reply(200, {
+          message: 'Not Found',
+          documentation_url: 'https://docs.github.com/rest/reference/users#get-a-user',
+        })
+        .get('/users/-')
+        .reply(200, {
+          id: 75544,
+          type: 'User',
+        })
+        .get('/users/test1-test-test')
+        .reply(200, {
+          message: 'Not Found',
+          documentation_url: 'https://docs.github.com/rest/reference/users#get-a-user',
+        })
+
+        // get permissions
+        .get('/repos/gitpoap/gitpoap-bot-test-repo/collaborators/gitpoap/permission')
+        .reply(200, {
+          user: {
+            permissions: {
+              admin: true,
+            },
+          },
+        })
+
+        // Test that a comment is posted with the correct body
+        .post(
+          '/repos/gitpoap/gitpoap-bot-test-repo/issues/25/comments',
+          issueCreatedBodyForValidUsers,
```

Comment:
hm - is this correct? more specifically, i thought if there are no valid usernames, then no comment should be made?

Reply:
`@gitpoap-bot @test-test- @test-test-- @-@test1-test-test@test2`

Here is one valid mention, 
`@-`